group.py

```diff
@@ -2,7 +2,7 @@
 
 # Your code to go here...
 
-data_structure = [
+my_group = [
     {"name": "Tom", "age": 12, "job": "developer", "relation": "Gils friend"},
```

Comment:
Instead of making "relation" a single string, you could make it a list, tuple, or another dictionary that separates the name of the relation (Gil) and the type of relation (friend)

Reply:
Agree! It will be hard to query this data structure.